app/src/main/java/org/oppia/android/app/options/AudioLanguageFragmentPresenter.kt

```diff
@@ -47,6 +48,13 @@ class AudioLanguageFragmentPresenter @Inject constructor(
       (fragment.activity as AudioLanguageActivity).finish()
     }
 
+    languageSelectionViewModel.selectedLanguage.observe(
+      fragment,
+      Observer {
+        Log.d("selectedLanguage", "selected language -> $it")
+      }
+    )
```

Comment:
Is this still needed? It seems to be debug only--can we remove it?

Reply:
I have removed it